cmake/Platform/Linux/Install_linux.cmake

```diff
@@ -9,14 +9,18 @@
 #! ly_setup_runtime_dependencies_copy_function_override: Linux-specific copy function to handle RPATH fixes
 set(ly_copy_template [[
 function(ly_copy source_file target_directory)
-    file(COPY "${source_file}" DESTINATION "${target_directory}" FILE_PERMISSIONS @LY_COPY_PERMISSIONS@ FOLLOW_SYMLINK_CHAIN)
+    set(full_target_directory "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${target_directory}")
```

Comment:
Does the DESTDIR env end with a trailing slash?

I think it would be safer to use `cmake_path(SET)` here to smartly concatenate the paths

Reply:
yes, when defined, it has a trailing slash and its the same way cmake deals with it: https://github.com/Kitware/CMake/search?q=%24ENV%7BDESTDIR%7D

I can try to change this to use cmake_path, something like this, right?:
```
cmake_path(APPEND full_target_directory $ENV{DESTDIR} ${CMAKE_INSTALL_PREFIX} ${target_directory})
```

Reply:
Yes, I was thinking of logic like the line you posted should work